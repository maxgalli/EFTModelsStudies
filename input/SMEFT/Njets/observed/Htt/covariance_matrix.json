{
    "r_Njets_0": {
        "r_Njets_0": 0.23497204144567943,
        "r_Njets_1": 0.026153877567108116,
        "r_Njets_2": 0.00485156744602163,
        "r_Njets_3": 0.007243185780405997,
        "r_Njets_4": 0.006372558889200992
    },
    "r_Njets_1": {
        "r_Njets_0": 0.026153877567108116,
        "r_Njets_1": 0.07515872559511286,
        "r_Njets_2": 0.0020928306597699166,
        "r_Njets_3": 0.0009750155096938154,
        "r_Njets_4": 0.009484494493000734
    },
    "r_Njets_2": {
        "r_Njets_0": 0.00485156744602163,
        "r_Njets_1": 0.0020928306597699166,
        "r_Njets_2": 0.048362108658217585,
        "r_Njets_3": -0.0013329072607546069,
        "r_Njets_4": -0.0028726303811196377
    },
    "r_Njets_3": {
        "r_Njets_0": 0.007243185780405997,
        "r_Njets_1": 0.0009750155096938154,
        "r_Njets_2": -0.0013329072607546069,
        "r_Njets_3": 0.1303660297071412,
        "r_Njets_4": -0.00905707742083727
    },
    "r_Njets_4": {
        "r_Njets_0": 0.006372558889200992,
        "r_Njets_1": 0.009484494493000734,
        "r_Njets_2": -0.0028726303811196377,
        "r_Njets_3": -0.00905707742083727,
        "r_Njets_4": 0.20549446709496194
    }
}