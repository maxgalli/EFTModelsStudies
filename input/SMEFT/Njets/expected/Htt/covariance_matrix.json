{
    "r_Njets_0": {
        "r_Njets_0": 0.22880653638125606,
        "r_Njets_1": 0.02249559250467781,
        "r_Njets_2": 0.00372351379250977,
        "r_Njets_3": 0.00716843139328305,
        "r_Njets_4": 0.005411437842271307
    },
    "r_Njets_1": {
        "r_Njets_0": 0.02249559250467781,
        "r_Njets_1": 0.07851408876398697,
        "r_Njets_2": 0.0018872635579297713,
        "r_Njets_3": 0.0012070629244618958,
        "r_Njets_4": 0.012611853526184878
    },
    "r_Njets_2": {
        "r_Njets_0": 0.00372351379250977,
        "r_Njets_1": 0.0018872635579297713,
        "r_Njets_2": 0.05501330674216749,
        "r_Njets_3": 0.002730662085219294,
        "r_Njets_4": -0.00024311731242827192
    },
    "r_Njets_3": {
        "r_Njets_0": 0.00716843139328305,
        "r_Njets_1": 0.0012070629244618958,
        "r_Njets_2": 0.002730662085219294,
        "r_Njets_3": 0.14910726108032268,
        "r_Njets_4": 0.004481827672897978
    },
    "r_Njets_4": {
        "r_Njets_0": 0.005411437842271307,
        "r_Njets_1": 0.012611853526184878,
        "r_Njets_2": -0.00024311731242827192,
        "r_Njets_3": 0.004481827672897978,
        "r_Njets_4": 0.25890923000904514
    }
}